content/docs/lists-and-keys.md

```diff
@@ -130,17 +130,17 @@ const todoItems = todos.map((todo, index) =>
 );
 ```
 
-We don't recommend using indexes for keys if the order of items may change. This can negatively impact performance and may cause issues with component state. Check out Robin Pokorny's article for an [in-depth explanation on the negative impacts of using an index as a key](https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318). If you choose not to assign an explicit key to list items then React will default to using indexes as keys.
+No recomendamos usar indices para keys si el orden de los items puede cambiar. Esto puede impactar negativamente el rendimiento y puede causar problemas con el estado del componente. Revisa el articulo de Robin Pokorny para una [explicacion en profundidad de los impactos negativos de usar un indice como key](https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318). Si eliges no asignar una key explicita a la lista de items, React por defecto usara indices como keys.
 
-Here is an [in-depth explanation about why keys are necessary](/docs/reconciliation.html#recursing-on-children) if you're interested in learning more.
+Aqui esta una [explicacion en profundidad sobre porque las keys son necesarias](/docs/reconciliation.html#recursing-on-children) si estas interesado en aprender mas.
 
-### Extracting Components with Keys
+### Extraccion de Components con Keys
 
-Keys only make sense in the context of the surrounding array.
+Las keys solo tienen sentido en el contexto de un array circundante.
 
-For example, if you [extract](/docs/components-and-props.html#extracting-components) a `ListItem` component, you should keep the key on the `<ListItem />` elements in the array rather than on the `<li>` element in the `ListItem` itself.
+Por ejemplo, si [extraes](/docs/components-and-props.html#extracting-components) un componente `ListItem`, debes mantener la key en el elemnto `<ListItem />` en el array en lugar de en el elemento `<li>` en el `ListItem` en si mismo.
```

Comment:
I think this can be better translated this way:
> ... debes mantener la key en los elementos `<ListItem />`  en el array en lugar de en los elementos `<li>` en el `ListItem` en sí.

Regardless of that, note `elemnto` before `<ListItem />`, it should be `elemento/s`
Note

Reply:
IMO 

> ...derberias mantener la key en los elementos `<ListItem />` en el array en lugar de en el elemento `<li>` en el `<ListItem />` en si mismo.

What do you think?